oap-server/server-receiver-plugin/receiver-proto/src/main/proto/envoy/service/accesslog/v2/als.proto

```diff
@@ -23,51 +20,6 @@ service AccessLogService {
   // API for "critical" access logs in which Envoy will buffer access logs for some period of time
   // until it gets an ACK so it could then retry. This API is designed for high throughput with the
   // expectation that it might be lossy.
-  rpc StreamAccessLogs(stream StreamAccessLogsMessage) returns (StreamAccessLogsResponse) {
-  }
-}
-
-// Empty response for the StreamAccessLogs API. Will never be sent. See below.
-message StreamAccessLogsResponse {
-}
-
-// Stream message for the StreamAccessLogs API. Envoy will open a stream to the server and stream
-// access logs without ever expecting a response.
-message StreamAccessLogsMessage {
-  message Identifier {
-    // The node sending the access log messages over the stream.
-    envoy.api.v2.core.Node node = 1 [(validate.rules).message.required = true];
-
-    // The friendly name of the log configured in :ref:`CommonGrpcAccessLogConfig
-    // <envoy_api_msg_config.accesslog.v2.CommonGrpcAccessLogConfig>`.
-    string log_name = 2 [(validate.rules).string.min_bytes = 1];
-  }
-
-  // Identifier data that will only be sent in the first message on the stream. This is effectively
-  // structured metadata and is a performance optimization.
-  Identifier identifier = 1;
-
-  // Wrapper for batches of HTTP access log entries.
-  message HTTPAccessLogEntries {
-    repeated envoy.data.accesslog.v2.HTTPAccessLogEntry log_entry = 1
-        [(validate.rules).repeated .min_items = 1];
-  }
-
-  // [#not-implemented-hide:]
-  // Wrapper for batches of TCP access log entries.
-  message TCPAccessLogEntries {
-    repeated envoy.data.accesslog.v2.TCPAccessLogEntry log_entry = 1
-        [(validate.rules).repeated .min_items = 1];
-  }
-
-  // Batches of log entries of a single type. Generally speaking, a given stream should only
-  // ever include one type of log entry.
-  oneof log_entries {
-    option (validate.required) = true;
-
-    HTTPAccessLogEntries http_logs = 2;
-
-    // [#not-implemented-hide:]
-    TCPAccessLogEntries tcp_logs = 3;
+  rpc StreamAccessLogs(stream envoy.service.accesslog.v3.StreamAccessLogsMessage) returns (envoy.service.accesslog.v3.StreamAccessLogsResponse) {
```

Comment:
Let's add comments on all our v2/* proto to explain why this is not an official v2.